dandiapi/api/models/version.py

```diff
@@ -60,6 +72,16 @@ def name(self):
     def size(self):
         return self.assets.aggregate(size=models.Sum('blob__size'))['size'] or 0
 
+    @property
+    def valid(self) -> bool:
+        if self.status != Version.Status.VALID:
+            return False
+
+        # Import here to avoid dependency cycle
+        from .asset import Asset
+
+        return not self.assets.filter(~models.Q(status=Asset.Status.VALID)).exists()
```

Comment:
A comment would be good, since this is slightly confusing to read at first glance
```suggestion
        # Return false if any of this version's assets have a status other than `VALID`
        return not self.assets.filter(~models.Q(status=Asset.Status.VALID)).exists()
```